docs/dev/build-instructions/build_linux.md

```diff
@@ -160,6 +160,15 @@ For all available build options, have a look [here](cmake_options.md).
 E.g. if you only want to build the server, use `cmake -Dclient=OFF ..`.
 
 
+## Using the BUILD_NUMBER variable
+
+We are currently using a [versioning scheme](https://www.mumble.info/blog/new-versioning-scheme/) that increases the "build" or "patch"
+component of the version string on every commit. This build number is specified via cmake as a variable e.g. ``-DBUILD_NUMBER=123``.
+Since we are exposing the complete version string in the Mumble UI as well as over the network protocol, it would be good if the ``BUILD_NUMBER``
+variable is set correctly when building the binaries. This is especially important for **distribution package maintainers**.
+If not specified, the version string will end in ``.0`` which is not breaking functionality but is also not ideal.
```

Comment:
Not true (to my knowledge).

We're using semantic versioning (more or less) meaning any change in the patch number does not reflect any functional changes.

The only reason we have the monotonically increasing build number is to ensure newer versions always get installed over older ones. It should not enable or disable any functionality on it's own. Therefore, it shouldn't matter if the `BUILD_NUMBER` is zero, unless you are packaging Mumble.

Reply:
Yes, I wrote it does **not** break functionality. However, I still think we want as much people as possible building Mumble with the patch component. This helps us when we get bug reports, usage statistics, or when we want to survey client/server versions of public servers.

I have updated the text to better reflect what I mean.